azurerm/resource_arm_managed_disk.go

```diff
@@ -149,6 +149,8 @@ func resourceArmManagedDiskCreateUpdate(d *schema.ResourceData, meta interface{}
 		skuName = compute.StandardLRS
 	} else if strings.EqualFold(storageAccountType, string(compute.StandardSSDLRS)) {
 		skuName = compute.StandardSSDLRS
+	} else if strings.EqualFold(storageAccountType, string(compute.UltraSSDLRS)) {
```

Comment:
🤔 

Is there a reason this entire if/else block isn't just:
```go
		skuName = string(compute.storageAccountType)
```

Reply:
Good point, I was just mimicking the existing structure, but I am happy to change it.

Reply:
In talking with the team it seems like there is a reason behind this, as we are going to release later today i'm just going to merge it in as is and can revisit it at a later time